src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import com.opencsv.CSVWriter;
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FileCsvService;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.FileCsvServiceImpl;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.strategy.handler.ActivityHandler;
+import core.basesyntax.service.strategy.handler.BalanceActivityHandler;
+import core.basesyntax.service.strategy.handler.PurchaseActivityHandler;
+import core.basesyntax.service.strategy.handler.ReturnActivityHandler;
+import core.basesyntax.service.strategy.handler.SupplyActivityHandler;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FruitDao fruitDao = new FruitDaoImpl();
+        FileCsvService fileService = new FileCsvServiceImpl();
+        Map<Operation, ActivityHandler> activityHandlerMap = new HashMap<>();
+        activityHandlerMap.put(Operation.BALANCE, new BalanceActivityHandler(fruitDao));
+        activityHandlerMap.put(Operation.SUPPLY, new SupplyActivityHandler(fruitDao));
+        activityHandlerMap.put(Operation.PURCHASE, new PurchaseActivityHandler(fruitDao));
+        activityHandlerMap.put(Operation.RETURN, new ReturnActivityHandler(fruitDao));
+        List<String[]> list = getStock();
+        try (CSVWriter csvFile = fileService.createCsvFile("stock.csv")) {
+            csvFile.writeAll(list);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't close file " + e);
+        }
+        FruitService fruitService = new FruitServiceImpl(fileService, activityHandlerMap);
```

Comment:
main method is not implemented according to task
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/cdc079c8-bc8e-49f6-9be6-56d6e80ae6c1)